src/main/kotlin/thatline/localup/chatgpt/service/ChatGptService.kt

```
@@ -41,26 +52,200 @@ class ChatGptService(
+                }.joinToString("\n")}
+                """
+            } else {
+                "\n[이달 예정된 주요 행사가 없습니다]"
+            }
+
+            """
+
+            [${business.address} 지역 행사/이벤트 정보]$mainEventInfo$upcomingEventsInfo
+            """.trimIndent()
+
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)